tests/python/contrib/test_hexagon/test_parallel_hvx_load_vtcm.py

```diff
@@ -308,7 +308,12 @@ def setup_and_run(hexagon_session, sch, a, b, c, operations, mem_scope="global")
     a_hexagon = tvm.runtime.ndarray.array(a, device=hexagon_session.device, mem_scope=mem_scope)
     b_hexagon = tvm.runtime.ndarray.array(b, device=hexagon_session.device, mem_scope=mem_scope)
     c_hexagon = tvm.runtime.ndarray.array(c, device=hexagon_session.device, mem_scope=mem_scope)
-    timer = module.time_evaluator("__tvm_main__", hexagon_session.device, number=100, repeat=10)
+
+    # These are reduced for CI but number=100 and repeat=10 does a good job of removing noise. 
+    number = 1 
```

Comment:
you could also condition these so they're reduced only in CI (`tvm.testing.utils.IS_IN_CI` is a flag thats only `True` when CI runs the tests)

Reply:
Ooooh I will use this from now on thats super useful!